src/planwise/client/mapping.cljs

```diff
@@ -45,7 +45,7 @@
 ;; only "classic" mapbox styles can be used with this static API.
 ;; the new API does not allow defining an overlay as a query parameter.
 (defn static-image [geojson]
-  (fmt/format "https://api.mapbox.com/v4/%s/geojson(%s)/auto/256x144.png?access_token=%s"
+  (fmt/format "https://api.mapbox.com/v4/%s/geojson(%s)/auto/256x158.png?access_token=%s"
```

Comment:
I just realized we can always use the `style` property and set the card `height` from the Reagent component. That way the height is a namespace `def` which can be used to create this URL and set the style. I think that might mess up the layout, but, it should be fixable using flex.